2021/ashok2022re/openreview/packages.tex

```diff
@@ -0,0 +1,27 @@
+% Import packages here
+\setcounter{secnumdepth}{3} % default value for 'report' class is "2"
+
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{makecell}
+\usepackage{textcomp}
+\usepackage{graphicx}
+
+\usepackage[normalem]{ulem}
+\newif{\ifhidecomments}
+
+\usepackage{graphicx}
+\usepackage{capt-of}% or \usepackage{caption}
+\usepackage{booktabs}
+\usepackage{varwidth}
+\usepackage{subfig}
+\usepackage{caption,stackengine}
+
```

Comment:
Please add the package `multirow` as you are using it in your tables:

```
\usepackage{multirow}
```

Reply:
Done! @koustuvsinha